meta-cli/src/utils/shell_words.rs

```diff
@@ -0,0 +1,153 @@
+// no-auto-license-header
+
+/*
+The MIT License (MIT)
+
+Copyright (c) 2020 Alexandre Piel
+Copyright 2018-2022 the Deno authors.
```

Comment:
Wrong license? copy/paste the integrality of the author's license file content in here, then it should be ready to go 👍